apidoc/Titanium/UI/WebView.yml

```diff
@@ -432,6 +432,48 @@ methods:
     returns:
         type: BackForwardList
 
+  - name: createPDF
+    summary: |
+        Create a PDF document representation from the web page currently displayed in the WebView.
+    description: |
+        If the data is written to a file the resulting file is a valid PDF document.
```

Comment:
We may want to include an example in the docs showing how to call this method and then write the blob to a file in the callback.